src/sdk/pynni/nni/compression/torch/utils/counter.py

```diff
@@ -21,15 +21,14 @@ def count_flops_params(model: nn.Module, input_size, verbose=True):
     the remained filters according to its mask, which
     not taking the pruned input channels into consideration,
     so the calculated FLOPs will be larger than real number.
-
     Parameters
     ---------
     model : nn.Module
         target model.
     input_size: list, tuple
         the input shape of data
-
-
+    custom_ops: dict
```

Comment:
Suggest to document "dict of what"

Reply:
agree, @colorjam suggest to give detailed example of how `custom_ops` looks like.

Reply:
I mentioned the mapping below, and add a reference to ``custom_mask_ops``. Please review the latest version, thanks~